### ISSUE TITLE
Inverted list displays mirrored on Android

### ISSUE THREAD

**Issue by tomazahlin** [650Industries/react-native-invertible-scroll-view#33]
It worked before our react-native upgrade. Now with react native 0.37.0, the inverted list displays mirrored upside down, when using onRefresh property on the list. It looks fine on iOS, but looks like there were some changes on Android.

Anyone else having the same problem?

**Comment by ptomasroos** [comment]
Yup.

**Comment by chris-pikul** [comment]
Can confirm this is an issue as of at least RN@0.36. RefreshConrol seems to be the affecting code.

**Comment by arthurjamain** [comment]
Removing the refreshcontrol indeed temporarily fixed the issue for me.

**Comment by jpshelley** [comment]
With the FlatList, SectionList, VirtualizedList the RefreshControl is included regardless, is there any thoughts as to why this is happening?

Should we attempt to fix the mirror issue in this library or in React Natives's FlatList alternatives?

**Comment by jpshelley** [comment]
This fixed the issue for me, but I'm not sure about the performance hit that a transform occurs https://github.com/expo/react-native-invertible-scroll-view/issues/41#issuecomment-291486104

edit: Looking into it more, it may seem that this would then perform the transform twice as the InvertedScrollView itself performs a transom 🤔 might need more investigation

**Comment by jcharbo** [comment]
@jpshelley doesn't the regular version of this control preform the transform twice?  Reading the code it appears to me that it flips the scroll and then flips each child.  I think comment #41 just does the same thing.

**Comment by jpshelley** [comment]
@jcharbo correct, thats what I was trying to say. I think a better fit would be to submit a PR that checks if OS is Android and doesn't perform the transform on all the children.

**Comment by jcharbo** [comment]
@jpshelley what's the difference between android and ios here?

**Comment by jpshelley** [comment]
@jcharbo Notice the original comment on this issue. The mirror bug is only relevant to Android for whatever reason due to the transom y style.

**Comment by jcharbo** [comment]
I see, I think we're actually talking about two different things.  I'm trying to make flatlist work with this and that is flipped for me on both IOS and android.
